src/extension/serializer.ts

```diff
@@ -870,3 +881,8 @@ export class GrpcSerializer extends SerializerBase {
     return this.plainCache.get(cacheId)
   }
 }
+
+function isGhostCell(cell: NotebookCellData): boolean {
```

Comment:
must utility methods on the Serializer are static methods. use same pattern for consistency?

Reply:
Done.